lib/models/tts.dart

```diff
@@ -21,10 +35,19 @@ class TtsModel {
     });
   }
 
+  String getMsgAuthor(String message) {
+    final parts = message.split(' ');
+    return parts[0];
+  }
+
   void speak(String message) {
```

Comment:
Might be better to pass the author along with the message and construct the spoken message in `TtsModel`?